docs/core/tools/dotnet-build.md

```diff
@@ -98,7 +100,7 @@ The project or solution file to build. If a project or solution file isn't speci
 
 * **`-o|--output <OUTPUT_DIRECTORY>`**
 
-  Directory in which to place the built binaries. You also need to define `--framework` when you specify this option. If not specified, the default path is `./bin/<configuration>/<framework>/`.
+  Directory in which to place the built binaries. If not specified, the default path is `./bin/<configuration>/<framework>/`.
```

Comment:
Having to define `--framework` was never true?

Reply:
I don't think so.  I'm not sure where that line came from.

Reply:
-o is basically broken for multi-targeted projects :(, it sets OutputPath as global property IIRC, and each TFM then overwrites the other :( I'm guessing this was added to avoid that case. We really should fix it instead.

Reply:
OK, I've added this:

> For projects with multiple target frameworks (via the `TargetFrameworks` property), you also need to define `--framework` when you specify this option.